docs/src/index.md

```diff
@@ -45,3 +45,93 @@ interpolate1d!(ftargetcols, xsourcecols, xtargetcols, fsourcecols, Linear(), Fla
 ```
 
 The above examples can be run on NVIDIA GPUs by setting `DA = CuArray`.
+
+## BilinearInterpolation
+`interpolatebilinear!` function can be used to perform bilinear interpolation on a single level on or a multiple horizontal levels. Both single threaded CPU and NVIDIA GPU platforms are supported. Examples for single level and multiple level cases are presented below.
+
+### Example
+
+Example for using bilinear interpolation a single level on a CPU:
```

Comment:
Before diving into the example, it could be helpful to give some background about the notations used, e.g. this interpolation is from a source grid of dimension `nsourcex` x `nsourcey` to a target grid of dimension `ntargetx` x `ntargety`